apps/opensearch/sycamore-opensearch.sh

```diff
@@ -320,14 +322,29 @@ create_certificates() {
         chmod 600 "data/${X}"
         ln -sfn "../data/${X}" "config/${X}"
     done
+
+    # 4. Use provided certificates for HTTP if possible
+    if [[ -f ${ARYN_ETC}/hostcert.pem && -f ${ARYN_ETC}/hostkey.pem ]]; then
```

Comment:
Can we check for presence of provided cert before we do anything related to using self-sign certs?

Reply:
No.  We need self-signed certs for node and admin.